CHANGES.rst

```diff
@@ -31,10 +31,16 @@ Unreleased
 * The parameter ``text`` for methods
   :meth:`.SubredditLinkFlairTemplates.update` and
   :meth:`.SubredditRedditorFlairTemplates.update` is no longer required.
+* Introduced a data class, :class:`.RedditErrorItem`, to represent an individual
+  error item returned from Reddit.
+* Class :class:`.RedditAPIException` now serves as a container for the
+  :class:`.RedditErrorItem`\ s. You can obtain the individual items by either
+  iterating over the :class:`.RedditAPIException` or using index numbers.
+* ``APIException`` is an alias to :class:`.RedditAPIException`.
 
 **Removed**
 
-* Converting :class:`.APIException` to string will no longer escape unicode
+* Converting :class:`.RedditAPIException` to string will no longer escape unicode
```

Comment:
This is a tricky one, but I think anyone reading this would care about the behavior on `APIException`.

Reply:
I'll make sure to keep the old version.